gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/inspection/inspectResponses.ts

```diff
@@ -0,0 +1,46 @@
+import { Result } from '@chainlink/gauntlet-core'
+import { SolanaCommand, TransactionResponse } from '@chainlink/gauntlet-solana'
+import { PublicKey } from '@solana/web3.js'
+import { CONTRACT_LIST, getContract } from '../../../../lib/contracts'
+import { inspect, Inspection } from '../../../../core/inspection'
+
+type Input = {}
+
+export default class OCR2InspectResponses extends SolanaCommand {
+  static id = 'ocr2:inspect:responses'
+  static category = CONTRACT_LIST.OCR_2
+
+  makeInput = (userInput): Input => {
+    if (userInput) return userInput as Input
+
+    return {}
+  }
+
+  constructor(flags, args) {
+    super(flags, args)
+
+    this.require(!!this.flags.state, 'Please provide flags with "state""')
+  }
+
+  execute = async () => {
+    const ocr2 = getContract(CONTRACT_LIST.OCR_2, '')
+    const program = this.loadProgram(ocr2.idl, ocr2.programId.publicKey.toString())
+
+    const state = new PublicKey(this.flags.state)
+    const input = this.makeInput(this.flags.input)
+    const data = await program.account.state.fetch(state)
+
+    const inspections: Inspection[] = []
+
+    const successfulInspection = inspect(inspections)
```

Comment:
correct me if i'm wrong, but `ocr2:inspect:responses` doesn't look like it does anything?

for EVM i think responses are usually fetched from events, but Solana doesn't have the same thing...
additionally, when the report is put on-chain it only has the median answer (no set of responses from each node)

Reply:
Yeah, for now I left it indicated only until we have some feed running, so I have an example with valid transmissions

Reply:
If you want something preliminary, you could try using the state from: https://github.com/smartcontractkit/chainlink-solana/blob/feature/relay/pkg/solana/contract_test.go#L70

it's from one of my local test runs

Reply:
up to you, otherwise it might be worth leaving a TODO in a comment :)

Reply:
Created a story for it. Will leave it for later